src/test/java/com/google/sps/data/DatastoreScheduledInterviewTest.java

```diff
@@ -0,0 +1,155 @@
+          (long) -1,
+          new TimeRange(
+              Instant.parse("2020-07-06T17:00:10Z"), Instant.parse("2020-07-06T18:00:10Z")),
+          "user@company.org",
+          "user@mail.com");
+
+  private final ScheduledInterview scheduledInterview2 =
+      ScheduledInterview.create(
+          (long) -1,
+          new TimeRange(
+              Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+          "user@company.org",
+          "user2@mail.com");
+  private final ScheduledInterview scheduledInterview3 =
+      ScheduledInterview.create(
+          (long) -1,
+          new TimeRange(
+              Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+          "user3@company.org",
+          "user2@mail.com");
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig()
+              .setDefaultHighRepJobPolicyUnappliedJobPercentage(0));
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Test whether the scheduledInterview was added to datastore.
+  @Test
+  public void createsAndStoresEntity() {
+    tester.create(scheduledInterview1);
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview storedScheduledInterview = tester.entityToScheduledInterview(entity);
+    ScheduledInterview copyScheduledInterview1 =
+        ScheduledInterview.create(
+            storedScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T17:00:10Z"), Instant.parse("2020-07-06T18:00:10Z")),
+            "user@company.org",
+            "user@mail.com");
+    assertEquals(storedScheduledInterview, copyScheduledInterview1);
+  }
+
+  // Tests whether all scheduledInterviews for a particular user are returned.
+  @Test
+  public void getsAllScheduledInterviews() {
+    tester.create(scheduledInterview1);
+    tester.create(scheduledInterview2);
+    tester.create(scheduledInterview3);
+    List<ScheduledInterview> result = tester.getForPerson("user@company.org");
+    ScheduledInterview copyScheduledInterview1 =
+        ScheduledInterview.create(
+            result.get(0).id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T17:00:10Z"), Instant.parse("2020-07-06T18:00:10Z")),
+            "user@company.org",
+            "user@mail.com");
+    ScheduledInterview copyScheduledInterview2 =
+        ScheduledInterview.create(
+            result.get(1).id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user2@mail.com");
+
+    assertEquals(2, result.size());
+    assertEquals(copyScheduledInterview1, result.get(0));
+    assertEquals(copyScheduledInterview2, result.get(1));
```

Comment:
Is there a benefit to Assert.assertEquals vs just assertEquals?

Reply:
Aim for only one assert statement per test. So here combine the expected into a list or whatever the result is.

Reply:
Don't think there's a difference but I see you use Assert.assertEquals so I'll change them in mine for consistency